### DISCUSSION TITLE
WIN10

### DISCUSSION THREAD

**Discussion by aalorart** [PhlexPlexico/G5API#292] [Q&A]
Can i install g5api on Windows? Cuz i can't find any installation guide for win10

**Comment by PhlexPlexico** [comment]
Hi,

Yes you can! You can use Docker for windows and use the same commands for docker compose.

Alternatively there is a way of installing all the components on windows (MariaDB, NodeJS, Caddy/Nginx/Apache). 

The instructions should largely be the same, just how you install said tools would be different. I would still *highly* recommend you use Docker though as it's the easiest installation method.

**Reply by aalorart** [reply]
Oh, thank you <3
